### PR TITLE
New bid adapter for OPEN8

### PR DESCRIPTION
**Pull request by aiichiro**
## Type of change

- [x] New bidder adapter   

## Description of change

- test parameters for validating bids
```
{
    bidder: 'open8',
    params: {
        slotKey: '2ae5a533'
    }
}
```
- contact email of the adapter’s maintainer: tdd-adtech@open8.com
- [x] official adapter submission

### CODE REVIEW

**Comment by jsnellbaker** [comment]
Also - can you please put together a docs PR ([here](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) for your bidder?  You can refer to the other bidder files for reference.

**Comment by aiichiro** [comment]
@jsnellbaker Thank you for your review.
Sorry, our server is not available now because it is under development yet.
But we tested adapter in staging environment and it worked.

I think we should send PR after our server has been available.
I'm sorry to trouble you and I was wondering if you could check PR after it.
Should we close this PR?

Also we create a doc PR.
https://github.com/prebid/prebid.github.io/pull/1142

Thanks

**Comment by jsnellbaker** [comment]
@aiichiro Thanks for clarifying.  Do you have a rough ETA for when the server would be available for use?  If it's in the relatively near-future, we could just keep this PR open for a bit.  If it's longer, we can close for now and resubmit it later.  Let me know either way.  Thanks.

**Comment by aiichiro** [comment]
@jsnellbaker Thanks for checking.
Now our server get to be available for use. 
I would appriciate it if you could check again.

By the way, our video is floating header overlay. so video size depends on window size.

Thanks

File: modules/open8BidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    } else if (ad.adType === AD_TYPE.BANNER) {
+      const bannerAd = bidderResponse.ad.banner;
+      Object.assign(bid, {
+        width: bannerAd.w,
+        height: bannerAd.h,
+        ad: bannerAd.adm,
+        mediaType: BANNER
+      });
+      try {
+        bannerAd.imps.forEach(impTrackUrl => {
```
Comment (jsnellbaker) [inline modules/open8BidAdapter.js]: Could add a check to verify the `bannerAd.imps` is populated before running the `forEach`.

File: modules/open8BidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+            type: 'image',
+            url: sync
+          });
+        });
+      }
+    }
+    return syncs;
+  },
+  onBidWon: function(bid) {
+    const winUrl = bid.nurl.replace(
```
Comment (jsnellbaker) [inline modules/open8BidAdapter.js]: Could add a check here to verify the `nurl` property is populated in the `bid` object.

**Comment by aiichiro** [comment]
@jsnellbaker Thanks for your review.
The code that you pointed out were problem exactly and I fixed it. Thank you.
Could you check them?
Thanks

**Comment by jsnellbaker** [comment]
Thanks for the update; LGTM

**Pull request merged**